demos/memes-elastic/README.md

```diff
@@ -1,7 +1,24 @@
-# Memes Elastic Demo
+# Elastic Demo: Analyzing Internet News
 
 # Setup
 
+This demo loads a data set of Internet news snippets from April 2009, collected by Memetracker (see [data source](#data-source)), into ElasticSearch, then uses Juttle with elastic adapter to read this data, search the news, compute aggregations, and visualize in the browser. The demo uses docker containers and maps local Juttle programs to be run inside the `outrigger` container.
+
+## Prerequisites
+
+Run docker machine:
+
+```
+docker-machine start default
```

Comment:
So docker-machine is a thing and I'm going to try it out on my linux too: https://docs.docker.com/machine/ as its used for managing docker instances on your local setup or on a cloud provider.